sql/cagg_api.sql

```diff
@@ -0,0 +1,162 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- Get information about the materialization table and bucket width.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.get_materialization_info(
+    continuous_aggregate REGCLASS
+) RETURNS RECORD AS
+$body$
+DECLARE
+  info RECORD;
+BEGIN
+    SELECT mat_hypertable_id AS materialization_id,
+           bucket_width::interval AS bucket_width
+      INTO info
+      FROM _timescaledb_catalog.continuous_agg
+      JOIN _timescaledb_catalog.continuous_aggs_bucket_function
```

Comment:
```suggestion
      FROM _timescaledb_catalog.continuous_agg,
      LATERAL _timescaledb_functions.cagg_get_bucket_function_info(mat_hypertable_id)
```
The metadata table `_timescaledb_catalog.continuous_aggs_bucket_function` is useless and will be removed.